packages/mlmpfr/mlmpfr.4.0.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name:         "mlmpfr"
+version:      "4.0.0"
+maintainer:   "Laurent Thévenoux <laurent.thevenoux@gmail.com>"
+authors:      "Laurent Thévenoux <laurent.thevenoux@gmail.com>"
+homepage:     "https://github.com/thvnx/mlmpfr"
+bug-reports:  "https://github.com/thvnx/mlmpfr/issues"
+license:      "LGPL-3.0"
+dev-repo: "git+https://github.com/thvnx/mlmpfr.git"
+build: [
+  ["oasis" "setup"]
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove:  ["ocamlfind" "remove" "mlmpfr"]
+depends: [
+  "ocaml" {>= "4.04"}
+  "ocamlfind"
+  "oasis"
```

Comment:
Are these two just build dependencies?

Reply:
You need to specify `extra-files` and then opam will fetch them for you, then you have to make sure in your project that the extra files are actually used. If you grep for it in the opam-repository there are some examples, but I can't remember the names specifically.

Reply:
OK, thanks! I'll take a look at it.